app/views/petitions/_rejected_petition.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1>
+  <span class="heading-secondary">Rejected petition</span>
+  <%= @petition.title %>
```

Comment:
Is it better to have `petition` is a local variable explicitly passed through or just rely on it being available as `@petition`? I'm not sure